compiler/one-cmds/one-create-quant-dataset

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env bash
+''''export SCRIPT_PATH="$(cd "$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")" && pwd)" # '''
+''''export PY_PATH=${SCRIPT_PATH}/venv/bin/python                                       # '''
+''''test -f ${PY_PATH} && exec ${PY_PATH} "$0" "$@"                                     # '''
+''''echo "Error: Virtual environment not found. Please run 'one-prepare-venv' command." # '''
+''''exit 255                                                                            # '''
+
+# Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import h5py as h5
+import numpy as np
+import argparse
+import glob
+import os
+import onelib.utils as oneutils
+
+
+def get_parser():
+    """Create and return given the argument parser"""
+    parser = argparse.ArgumentParser(
+        description='command line tool to convert data files to hdf5 file')
+    parser.add_argument(
+        "-i",
+        "--input_type",
+        type=str,
+        help="Input type of either rawdata or numpy",
+        choices=['rawdata', 'numpy'],
+        required=True)
+    parser.add_argument(
+        "-o",
+        "--output_type",
+        type=str,
+        help="Output type to be converted",
+        choices=['hdf5'],
+        required=True)
```

Comment:
> can you make this work this option as hdf5 as default?

For now, Yes. :)

Not now, but considering the possibility of adding a new `output_type`, 
i added an option of `output_type` even if it only supports the converter of `hdf5` now.

Reply:
I mean, I want to use this tool without `--output_data_format`.
Maybe `required=False` will do.

Reply:
> Maybe required=False will do.

Okay. Thank you for your comment😄, 
I found `default` option in python's `argparse` and reflected the code as follows.
```
  parser.add_argument(
      "-o",
      "--output_data_format",
      type=str,
      help="Output data format to be converted (default: hdf5)",
      choices=['hdf5'],
      default='hdf5')
```
And additionally  I think it is necessary, so i added a `one-create-quant-dataset_003.test` file that tests whether `hdf5` is set well as a `default`.

Reply:
@Seunghui98 might have added the option because of my comment (https://github.com/Samsung/ONE/pull/12186#issuecomment-1842194414). When I suggested `one-create-quant-dataset`, I imagined this can be a unified tool that can make any format of dataset. That is why the name of this tool (`one-create-quant-dataset`) does not imply that its output format is hdf5 (unlike `rawdata2hdf5`). So, I think it is natural to have `output_data_format` option.

> I mean, I want to use this tool without --output_data_format.

For user convenience, setting a default value looks good to me.

Reply:
@jinevening , thanks for the explanation. Currently I want to know what would be the problem if we add this option in the future when needed. If there is no critical issue, I will be against adding this option at this moment.
Why? it looks odd to add an option with just only one selection. And cause of this, there is additional test needed.

Reply:
@seanshpark 

> Q) do we really need this option now?

Now only `hdf5` is supported, so it is not necessary now, 
but I think it is a necessary option because it is expandable like the comment(https://github.com/Samsung/ONE/pull/12186#issuecomment-1842194414, https://github.com/Samsung/ONE/pull/12186#discussion_r1418155804).

> what may be the problem if we add this option in the future if there is another output format?

I think we need to re-modify the code that adds the option in a similar structure as it is now.

Reply:
> it looks odd to add an option with just only one selection. And cause of this, there is additional test needed.

I understand. @seanshpark is a maintainer, so let's follow his opinion.